lib/identity-document.js

```diff
@@ -0,0 +1,94 @@
+const {spawn} = require('child_process');
+const temporary = require('temporary');
+const fs = require('fs');
+
+const AWS_PUBLIC_CERTIFICATE =
```

Comment:
I really like that you're building this into the code.  I think if this cert ever needed to change, the deployment overhead would justify the work.